docs/language_concepts/01_functions.md

```diff
@@ -86,3 +86,14 @@ follows:
 ```rust
 assert(MyStruct::sum(s) == 42);
 ```
+
+## Lambdas
+
+Lambdas are anonymous functions. They follow the syntax of rust - `|arguments| return_value`.
```

Comment:
Can we expand this syntax to `|arg1, arg2, ..., argN| return_expression` ?

Reply:
fix: https://github.com/noir-lang/docs/pull/356/commits/df0de5f7abc7bfc7ffca7a1944c62e433cc81ba9